### PR TITLE
Add ability to deploy the CRD to kubevirt's vagrant deployment.

### PR DESCRIPTION
**Pull request by mpolednik**

### CODE REVIEW

**Review by fabiand — APPROVED** [review]
This idea in generally looks good to me.

Please add a really nice README illustrating how to use it, and then I'd suggest to write a nice blog post about it.

Simple yet effective mechanism for persistence.

File: hack/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+KUBEVIRT="$GOPATH/src/kubevirt.io/kubevirt/"
+KUBECTL="$KUBEVIRT/cluster/kubectl.sh"
+
+# We need to grab absolute path to all manifests since we'll have to pushd
+# later on.
+MANIFESTS=$(find $(pwd)/manifests -name *.yaml)
```
Comment (petrkotas) [inline hack/deploy.sh:6]: This is a total mess, but I understand why we need this temporary solution.

**Review by petrkotas — APPROVED** [review]
Some minor comments, otherwise LGTM

File: hack/deploy.sh (hunk 1 above)
Comment (fabiand) [reply hack/deploy.sh:6]: @petrkotas make vagrant-deploy can probably be reduced to kubectl apply -f manifests/persistentvm-resource.yaml -- no need for hack-deploy

Comment (mpolednik) [reply hack/deploy.sh:6]: @fabiand that will be doable when followup PR is in as it routes cluster/kubectl.sh to kubevirt's installation

**Pull request merged**